osfoffline/application/background.py

```diff
@@ -1,103 +1,82 @@
 # -*- coding: utf-8 -*-
-import threading
 import asyncio
 import logging
+import threading
 
 from watchdog.observers import Observer
-import osfoffline.database_manager.models as models
-import osfoffline.polling_osf_manager.polling as polling
-import osfoffline.filesystem_manager.osf_event_handler as osf_event_handler
+
+from osfoffline.database_manager import models
 from osfoffline.database_manager.db import session
+from osfoffline.filesystem_manager import osf_event_handler
+from osfoffline.polling_osf_manager import polling
+
+
+logger = logging.getLogger(__name__)
 
 
 class BackgroundWorker(threading.Thread):
+
     def __init__(self):
         super().__init__()
+
+        # Start out with null variables for NoneType errors rather than Attribute Errors
         self.user = None
-        self.osf_folder = ''
```

Comment:
This change should help with a certain rather nasty bug that occurred when the path was no longer valid (and it just tried to save to `/OSF` instead); thanks.